src/share/timing/Makefile

```diff
@@ -7,7 +7,10 @@ VPATH := $(GPTL_DIR)
 ifndef MOD_SUFFIX
    MOD_SUFFIX := mod
 endif
-
+ifeq ($(SMP),TRUE)
```

Comment:
In cime's main makefile, we have:

```Makefile
ifeq ($(strip $(SMP)),TRUE)
```

Note the use of `strip`. Should this also use `strip` for consistency?

Reply:
sure, I've updated it to use strip